### ISSUE TITLE
Peer Review

### ISSUE THREAD

**Issue by wanxinglu** [mnercessian/ORIE4741proj#12]
This project aims to predict outcomes and point differential of NFL games using datasets containing game-by-game information from previous NFL games. They clean and modify data from 32 files for each NFL team and concatenate the datasets together. The team compiles 22 features for both home and away teams, including points per game, win percentages, and first downs per game. Then they split the data into 80:20 training set and utilized leasts squares regression to construct the model, and create multiple models using various subsets of the feature space and analysis the error of each model using MSE.

The team is very thoughtful in cleaning and modifying the data, for example adding unique identifiers and accounting for missing values by converting the data with a binary encoding. Also, the team recognizes that utilizing all 22 features overfits the models and plans to mitigate this issue in the future by limiting the collinearity of the features. Furthermore, the analysis of the models were very nuanced and discussed the noisiness of the results and true distribution.

One area of improvement would be to elaborate on the specific features chosen to fit the model. The terms are fairly technical and to someone who is not familiar with NFL terminology, it could be unclear how each feature affects the model. Also, the graphs in the appendix are small and it is difficult to see the axis and the labels. Lastly, the group does not incorporate any loss functions to their analysis, which could help with their final model analysis. For example, to predict outcomes of individual games, a zero-one loss function could help determine which models are performing best.
